pom.xml

```diff
@@ -90,6 +96,11 @@
         <!-- Old version until this issue is resolved: https://github.com/ImmobilienScout24/illegal-transitive-dependency-check/issues/30 -->
         <version>1.0.4</version>
       </dependency>
+      <dependency>
+        <groupId>org.jacoco</groupId>
+        <artifactId>org.jacoco.agent</artifactId>
+        <version>${version.jacoco.plugin}</version>
```

Comment:
I'd expect that jacoco is a compile/build/tool dependency, not a real normal dependency.
Is there any way we can restrict jacoco to one (or more) plugin classpaths?

Reply:
It makes sense, will look into it.